.github/workflows/tests.yml

```diff
@@ -57,6 +57,16 @@ jobs:
     - uses: ./.github/actions/setup-test-environment
     - name: Install JavaScript dependencies with Yarn
       run: yarn check || yarn install --frozen-lockfile;
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: yarn cache dir
+    - uses: actions/cache@v2
+      id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

Comment:
Huh, seeing an error here:
 
```
Run actions/cache@v2
  with:
    key: Linux-yarn-08fd7bab35005d3e8e92ed89e54f7a999ee6535e35e43d54deb4a9a5efd3bf37
    restore-keys: Linux-yarn-
  
Error: Input required and not supplied: path
```

Reply:
I'm a little stumped, to be honest, sorry! 

But, your `yarn cache dir` seems to have output something useful:

```
Run yarn cache dir
  yarn cache dir
  shell: /usr/bin/bash -e {0}
/home/runner/.cache/yarn/v6
```

Reply:
Ah so do we need to define an output of name `dir` for `yarn-cache-dir-path`?

Reply:
aha -- https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-composite-actions